config/schedule.rb

```diff
@@ -49,6 +49,10 @@ def local(time)
   end
 end
 
+every :day, at: local("07:00 am"), roles: [:cron] do
+  rake "appointment_notification:covid_medication_reminders[20000]"
```

Comment:
The rake argument syntax always looks weird to me.

Reply:
It is weird. And it gives ambiguous errors in zsh.